app/views/wishlists/show.html.erb

```diff
@@ -1,14 +1,28 @@
-<h1><%= @wishlist.name %></h1>
+<div class="container">
+  <h1 class="wishlist-name"><%= @wishlist.name %></h1>
 
-<p><%= @wishlist.description %></p>
+  <p class="wishlist-description"><%= @wishlist.description %></p>
 
-<p>
-  <strong>Your unique link for this whislist:</strong><br>
-  <%= wishlists_for_guest_url(@wishlist.token) %>
-</p>
+  <div class="input-group wishlist-url-display input-group-sm">
+    <span class="input-group-addon" id="sizing-addon2">Link for sharing:</span>
+    <input type="text" class="form-control" id="unique_url" value="<%= wishlists_for_guest_url(@wishlist.token) %>" aria-describedby="sizing-addon1">
```

Comment:
We don't care about accessibility anywhere else, why start here with `aria` attributes?